sympy/codegen/tests/test_algorithms.py

```diff
@@ -132,7 +132,7 @@ def test_newtons_method_function__ccode_parameters():
 x=     0.86547 d_x=  3.6902e-17
 """  # try to run tests with LC_ALL=C if this assertion fails
 
-
+@may_xfail
```

Comment:
The test occasionally yields values significantly greater or lesser than the desired result, as indicated by the condition: `abs(result - ref) < req`.

Reply:
Actually, this test is failing in fresh clone too.
https://github.com/sympy/sympy/blob/00277a0b347833a5f8872094c0109867aaa732ee/sympy/codegen/tests/test_algorithms.py#L136